packages/wdio-browserstack-service/tsconfig.json

```
@@ -8,5 +8,6 @@
   "include": [
     "src/**/*",
     "../../@types"
-  ]
+  ],
+  "types": [ "node" ],
```

I don't think this is needed.
```suggestion
  ]
```